docs/simpledroidble/usage.rst

```diff
@@ -9,39 +9,103 @@ This code is currently under active development and some features are not yet
 implemented or their API might change, but should be enough to help you get
 started.
 
+Obtain AAR dependencies
+=======================
+
+Using prebuilt AAR
+------------------
+
+1. Download the latest SimpleDroidBLE packages from the `releases page <https://github.com/OpenBluetoothToolbox/SimpleBLE/releases/>`_.
+
+2. Follow `Add SimpleDroidBLE to your project`_.
+
+Building from source
+--------------------
+
+1. Clone `the repo <https://github.com/OpenBluetoothToolbox/SimpleBLE>`_.
+
+2. Build the libraries using gradle (``./simpledroidble/gradlew assembleRelease``) or Android Studio. The following libraries will be built:
+    - ``simpledroidble/simpledroidble/build/outputs/aar/simpledroidble-release.aar`` 
+    - ``simpleble/src/backends/android/simpleble-bridge/build/outputs/aar/simpleble-bridge-release.aar``
+
+3. Follow `Add SimpleDroidBLE to your project`_.
+
+Add SimpleDroidBLE to your project
+==================================
+
+From AAR file
+-------------
+
+1. Place the package on your app's directory, and take note of its path. We'll use ``example-app/app/libs/{simpledroidble,simpleble-bridge}.aar``.
+
+2. Declare the dependency on your app's ``build.gradle{.kts}`` file (note: relative paths begin from your app's module folder)::
```

Comment:
I would have a separate copy of this specific for `simpleble-bridge` inside `simpleble/usage.rst`